apkid/rules/elf/obfuscators.yara

```diff
@@ -388,3 +388,21 @@ rule snapprotect : obfuscator
   condition:
     is_elf and 1 of ($a,$b)
 }
+
+rule safeengine : obfuscator
+{
+  meta:
+    description = "Safeengine"
```

Comment:
Maybe also add LLVM to the name as we do in Obfuscator-LLVM (`Safengine LLVM`). For the rest LGTM

Reply:
Ok.